ipv6-in-vnet-StdLB/README.md

```diff
@@ -0,0 +1,22 @@
+# Create a dual stack IPv4/IPv6 VNET with 2 VMs
```

Comment:
add a note here that mentions the feature must be registered in the sub before you can deploy the template